page.codeberg.JakobDev.jdPermissionStoreEdit.yaml

```diff
@@ -0,0 +1,25 @@
+id: page.codeberg.JakobDev.jdPermissionStoreEdit
+runtime: org.kde.Platform
+runtime-version: "6.8"
+sdk: org.kde.Sdk
+command: jdpermissionstoreedit
+finish-args:
+  - --talk-name=org.freedesktop.impl.portal.PermissionStore
+  - --filesystem=xdg-data/flatpak/db:ro
```

Comment:
these two requires exceptions. 

https://docs.flathub.org/docs/for-app-authors/linter#finish-args-portal-impl-component-talk-name

https://docs.flathub.org/docs/for-app-authors/linter#finish-args-unnecessary-xdg-data-subdir-mode-access

Reply:
why do you need direct access to the db itself?

Reply:
>  why do you need direct access to the db itself?

It is not possible to list all available tables using D-Bus. See flatpak/xdg-desktop-portal#1658.

> https://docs.flathub.org/docs/for-app-authors/linter#finish-args-portal-impl-component-talk-name

As his was added yestaerday, did you add that just for me?

Reply:
No planned to do it a while back but forgot about it then this submission reminded me.